src/components/Text/index.tsx

```diff
@@ -41,7 +41,7 @@ export const SmallText = styled.span`
 export const TinyText = styled.span`
   ${baseTextStyles};
   font-size: ${fontSize[14]};
-  color: ${color.foregroundSubtle};
+  color: ${props => props.color ?? color.foregroundSubtle};
```

Comment:
Why do we need this `color` prop? You can also use the styled component to overwrite this, right? This is not really consistent with the other text elements like H*, Text, etc.

Reply:
I think for a header that could be custom to any purpose with an icon and a color, it would make sense to have it like this so you can have it green, red, yellow or whatever color suits you best.